ts/@live-compositor/core/package.json

```diff
@@ -6,12 +6,16 @@
   "scripts": {
     "lint": "eslint .",
     "typecheck": "tsc --noEmit",
-    "watch": "tsc --watch",
+    "watch": "tsc --watch --preserveWatchOutput",
```

Comment:
Terminal will not be cleared after file is saved and errors/warnings are printed again. This is needed to stream `npm run output` from multiple commands